compose.yaml

```
@@ -0,0 +1,37 @@
+services:
+  gnodev:
+    build: ./contract/
+    ports:
+      - "8888:8888"
+      - "26657:26657"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8888/r/volos"]
+      interval: 5s
+      retries: 5
```

```suggestion
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8888/r/volos/render"]
      interval: 60s
      retries: 5
```

There seems to be some trouble with rendering /r/volos sometimes since it uses render package's `Render()` (this doesn't affect the healthcheck, but it keeps printing package not found which is a bit misleading). Also I think 5s healthcheck is too frequent, maybe set it to 30 or even 60 to reduce noise in the logs.